src/Tools/ProcessWatchdog/Program.cs

```diff
@@ -67,7 +70,8 @@ private int Run()
 
                         if (_options.Screenshot)
                         {
-                            ScreenshotSaver.SaveScreen(_options.Executable, _options.OutputFolder);
+                            string description = Path.GetFileNameWithoutExtension(_options.Executable);
```

Comment:
This fixes a bug that prevented screen shots from being written. The `ScreenShot` class uses the `description` parameter to construct the name of the saved screen shot file relative to the ProcessWatchdogOutput directory. We were using `options.Executable`, the path to the executable for which the screen shot was generated, as the description. But this path included directory name components, resulting in a file name with extra directory components (for example, the path to the screen file might come out as `ProcessWatchdogOutput\screenshot-Binaries\RunTests\RunTests.exe`). Those extra path components didn't exist under ProcessWatchdog output, so the attempt to save the screen shot file failed.